src/MatrixFields/matrix_shape.jl

```diff
@@ -3,19 +3,39 @@ struct Square <: AbstractMatrixShape end
 struct FaceToCenter <: AbstractMatrixShape end
 struct CenterToFace <: AbstractMatrixShape end
 
+"""
+   matrix_shape(matrix_field)
+
+Returns the matrix shape for matrix_field.
+"""
```

Comment:
It's better to avoid duplicating docstrings when possible. Could you modify the original docstring instead of adding new ones?

Reply:
I think I fixed, but please let me know if you want me to modify